frontend/app/services/subscription-service.server.ts

```diff
@@ -140,5 +140,40 @@ function createSubscriptionService() {
     return response;
   }
 
-  return { getSubscription, updateSubscription, validateConfirmationCode };
+  async function requestNewConfirmationCode(userEmail: string, userId: string) {
+    const auditService = getAuditService();
+    const instrumentationService = getInstrumentationService();
+
+    auditService.audit('alert-subscription.request', { userId });
```

Comment:
I think the event name should be more descriptive here. For example, what is the user requesting?

Reply:
Maybe `alert-subscription.request-confirmation-code`?